geode-core/src/test/java/org/apache/geode/management/bean/stats/MemberLevelStatsTest.java

```diff
@@ -339,6 +342,41 @@ public void testRegionCounters() {
     assertThat(memberMBeanBridge.getTotalPrimaryBucketCount()).isZero();
   }
 
+  @Test
+  public void testVMStats() {
+    Statistics[] realStats = statisticsManager.findStatisticsByType(VMStats50.getGCType());
+    long[] totals = modifyStatsAndReturnTotalCountAndTime(10, 2500, realStats);
+    memberMBeanBridge.addVMStats(statSampler.getVMStats());
+    assertThat(memberMBeanBridge.getGarbageCollectionCount()).isEqualTo(totals[0]);
+    assertThat(memberMBeanBridge.getGarbageCollectionTime()).isEqualTo(totals[1]);
+
+    long[] newTotals = modifyStatsAndReturnTotalCountAndTime(20, 3500, realStats);
+    sampleStats();
+    assertThat(memberMBeanBridge.getGarbageCollectionCount()).isEqualTo(newTotals[0]);
+    assertThat(memberMBeanBridge.getGarbageCollectionTime()).isEqualTo(newTotals[1]);
+  }
+
+  private long[] modifyStatsAndReturnTotalCountAndTime(
+      long baseCount, long baseTime,
+      Statistics[] modifiedStats) {
+    long[] totalCountAndTime = {0, 0};
+    for (Statistics gcStat : modifiedStats) {
+      StatisticDescriptor[] statistics = gcStat.getType().getStatistics();
+      for (StatisticDescriptor d : statistics) {
+        if ("collections".equals(d.getName())) {
```

Comment:
I think you should either reference a constant instead of referencing "collections" and "collectionTime" in multiple classes.